introduction.md

```diff
@@ -1,2 +1,3 @@
 Artem Trubitsyn
-JavaScript
\ No newline at end of file
+*JavaScript
+*Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?